src/main/resources/static/api.yaml

```diff
@@ -401,7 +401,7 @@ paths:
             application/xml:
               schema:
                 $ref: '#/components/schemas/ConsentArtefactRepresentation'
-  /consent-requests/{request-id}/consent-artefacts:
+  /consent-requests/{consent-request-id}/consent-artefacts:
```

Comment:
it could be just an id, IMO. but it's okay to have full variable name also.

Reply:
swagger was throwing an error, because I think it didn't comply with the spec. Its ok I believe